### PR TITLE
Feature + Curated Cards

### PR DESCRIPTION
**Pull request by skumawat1**
Fix # https://adobe-acs-group.atlassian.net/browse/WRLDBAEMS-46 ,  https://adobe-acs-group.atlassian.net/browse/WRLDBAEMS-44

Test URLs:
- Before:https://main--world-bank--aemsites.hlx.live/en/drafts/skumawat/curated-cards
- After: https://banner-cards--world-bank--aemsites.hlx.live/en/drafts/skumawat/curated-cards

author-url: https://author-p136806-e1403562.adobeaemcloud.com/ui#/@worldbank/aem/universal-editor/canvas/author-p136806-e1403562.adobeaemcloud.com/content/world-bank/global/en/drafts/skumawat/curated-cards.html?ref=banner-cards
Desktop View
image

Tab View
image

Mobile View

### CODE REVIEW

File: blocks/curated-cards/curated-cards.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  if (headingContent) {
+    heading.textContent = headingContent.textContent;
+  }
+
+  textWrapper.append(heading, tagElement);
+  li.append(imageDiv, textWrapper);
+
+  row.innerHTML = ''; // Clear the original row content
+
+  moveInstrumentation(row, li);
```
Comment (thomsebastin) [inline blocks/curated-cards/curated-cards.js:62]: Can be named into something meaningful.

File: blocks/curated-cards/curated-cards.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  return li;
+}
+
+// Main function to decorate the block
+export default function decorate(block) {
+  const ul = createElement('ul');
+  const childrenArray = Array.from(block.children);
```
Comment (thomsebastin) [inline blocks/curated-cards/curated-cards.js:69]: name this to something else meaningful.

**Review by thomsebastin — CHANGES_REQUESTED** [review]

File: blocks/curated-cards/curated-cards.css [hunk 3]
```diff
@@ -0,0 +1,187 @@
+.feature-card picture {
```
[the remark this answers is not in the document]
Comment (skumawat1) [reply blocks/curated-cards/curated-cards.css]: done

File: blocks/curated-cards/curated-cards.css [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  inset: 0;
+  box-sizing: border-box;
+  background: linear-gradient(1.27deg, #012740 5.95%, #0127400f 52.78% );
```
[the remark this answers is not in the document]
Comment (skumawat1) [reply blocks/curated-cards/curated-cards.css]: done

File: blocks/curated-cards/curated-cards.css [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+}
+
+.feature-card picture img {
+  object-fit: cover;
+  width: 1456px;
+  height: 700px;
+  z-index: -1;
```
[the remark this answers is not in the document]
Comment (skumawat1) [reply blocks/curated-cards/curated-cards.css]: yes this is required as we are overlaying image with this linear gradient style

File: blocks/curated-cards/curated-cards.css [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+  border-radius: 10px 10px 0 0;
+  overflow: hidden;
+}
+
+.feature-card-content {
+  position: absolute;
+  bottom: 20px; /* Position slightly up from the bottom */
+  left: 20px; /* Position slightly to the right from the left edge */
+  width: calc(100% - 40px); /* Adjust width to account for left/right padding */ 
+  color: #fff; /* Text color for contrast */
```
[the remark this answers is not in the document]
Comment (skumawat1) [reply blocks/curated-cards/curated-cards.css]: done

File: blocks/curated-cards/curated-cards.css [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+  padding: 20px;
+  box-sizing: border-box;
+}
+
+.feature-card-content-heading {
+  font-size: 48px;
```
[the remark this answers is not in the document]
Comment (skumawat1) [reply blocks/curated-cards/curated-cards.css]: done

File: blocks/curated-cards/curated-cards.css [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+  margin: 0 0 10px;
+  line-height: 58px;
+  font-weight: 700;
+}
+
+.feature-card-content-description {
+  font-size: 20px;
```
[the remark this answers is not in the document]
Comment (skumawat1) [reply blocks/curated-cards/curated-cards.css]: done

**Review by skumawat1 — COMMENTED** [review]
I have addressed all review comment

**Review by mkbansal1 — APPROVED** [review]

**Review by thomsebastin — APPROVED** [review]

**Pull request merged**